### PR TITLE
NetSuite 11.0

### PR DESCRIPTION
**Pull request by dhirsch1**

### CODE REVIEW

File: netsuite/11.0/modules/ROOT/pages/netsuite-connector-upgrade-migrate.adoc [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+[%header%autowidth.spread]
+|===
+| Operation |Changes
+| Change Email | Returns a NetSuiteSoapResult object in the output metadata
+| Change Password | Returns a NetSuiteSoapResult object in the output metadata
+| Get Account Governance Info | Returns a GetAccountGovernanceInfoResult object in the output metadata
+| Get Data Center URLs | Returns a NetSuiteSoapResult object in the output metadata.
+| Get Server Time | Returns a NetSuiteSoapResult object in the output metadata
+|===
```
Comment (tomastoloza) [inline netsuite/11.0/modules/ROOT/pages/netsuite-connector-upgrade-migrate.adoc:43]: Missing getIntegrationGovernanceInfo operation

File: netsuite/11.0/modules/ROOT/pages/netsuite-connector-upgrade-migrate.adoc [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+| Async Delete List Records | Removed this operation. For equivalent functionality, use the Async Delete List operation and specify the recordRef or baseRefType in the Message field of the input metadata.
+| Async Get List | Removed the Retrieved Base Ref Types field. You now specify the associated values in the Message field of the input metadata.
+| Async Get List Records | Removed this operation. For equivalent functionality, use the Get List operation and specify `FILE` as the record type in the Message field of the input metadata.
+| Async Initialize List | Removed the Initialize Records field. You now specify the associated values in the Message field of the input metadata.
+| Async Search | Removed the Search Record and Criteria fields. You now specify the associated values in the Message field of the input metadata.
+| Async Update List | Removed the Record Type and Records fields. You now specify the associated values in the Message field of the input metadata.
+| Async Upsert List | Removed the Record Type and Records fields. You now specify the associated values in the Message field of the input metadata.
+| Check Async Status | No changes.
+|===
```
Comment (tomastoloza) [inline netsuite/11.0/modules/ROOT/pages/netsuite-connector-upgrade-migrate.adoc:63]: Missing getAsyncResult operation

File: netsuite/11.0/modules/ROOT/pages/netsuite-connector-upgrade-migrate.adoc [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+| Operation |Changes
+| Add File | Removed this operation. For equivalent functionality, use the Add Record operation and specify `FILE` as the record type in the Message field of the input metadata.
+| Add List a|
+* Removed the Record Maps field. You now specify this value in the Message field of the input metadata.
+* Returns an addListResponse object in the palyload of the output metadata.
+| Add Record a|
+* Removed the Attributes field. You now specify the associated values in the Message field of the input metadata.
+* Returns a NetsuiteSoapResult object in the output metadata.
+| Add Record Objects a| Removed this operation. For equivalent functionality, use the Add Record operation.
```
Comment (tomastoloza) [inline netsuite/11.0/modules/ROOT/pages/netsuite-connector-upgrade-migrate.adoc]: Change equivalent operation
```suggestion
| Add Record Objects a| Removed this operation. For equivalent functionality, use the Add List operation.
```

File: netsuite/11.0/modules/ROOT/pages/netsuite-connector-upgrade-migrate.adoc [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+*  Returns a NetsuiteSoapResult object in the output metadata with an attached object as the payload..
+| Delete a|
+* Removed the Base Ref field. You now specify the associated values in the Message field of the input metatada.
+* Returns a deleteResponse object in the payload of the output metadata.
+| Delete List a|
+* Removed the Deleted Base Ref Types field. You now specify the
+ associated values in the Message field of the input metatada.
+* Returns a NetsuiteSoapResult object in the output metdata.
+| Delete Record | Removed this operation. For equivalent functionality, use the Delete operation.
+| Delete Records List | Removed this operation. For equivalent functionality, use the Delete operation.
```
Comment (tomastoloza) [inline netsuite/11.0/modules/ROOT/pages/netsuite-connector-upgrade-migrate.adoc]: Change equivalent operation
```suggestion
| Delete Records List | Removed this operation. For equivalent functionality, use the Delete List operation.
```

**Review by tomastoloza — CHANGES_REQUESTED** [review]

File: netsuite/11.0/modules/ROOT/pages/netsuite-connector-upgrade-migrate.adoc (hunk 1 above)
Comment (dhirsch1) [reply netsuite/11.0/modules/ROOT/pages/netsuite-connector-upgrade-migrate.adoc:43]: Does this return a GetIntegrationGovernanceInfoResult object?

Comment (tomastoloza) [reply netsuite/11.0/modules/ROOT/pages/netsuite-connector-upgrade-migrate.adoc:43]: Yes

**Comment by dhirsch1** [comment]
This PR was replaced.

**Pull request closed without merging**